pkg/virt-launcher/virtwrap/access-credentials/access_credentials_test.go

```diff
@@ -343,12 +339,14 @@ var _ = Describe("AccessCredentials", func() {
 		})
 
 		matched := false
-		mockConn.EXPECT().QemuAgentCommand(cmdSetPassword, domName).MinTimes(1).DoAndReturn(func(funcCmd string, funcDomName string) (string, error) {
-			if funcCmd == cmdSetPassword {
+		mockDomain.EXPECT().SetUserPassword(user, password, libvirt.DomainSetUserPasswordFlags(0)).MinTimes(1).DoAndReturn(
+			func(u, p string, flags libvirt.DomainSetUserPasswordFlags) error {
+				Expect(u).To(Equal(user))
+				Expect(p).To(Equal(password))
+				Expect(flags).To(Equal(libvirt.DomainSetUserPasswordFlags(0)))
```

Comment:
I wonder if we need these Expects at all since they should have been asserted by the definiton of the mock call already.

Reply:
+1
The signature of the mock expectation is enough

Reply:
Done.